scripts/supercloud/run_option_learning_experiments.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Looked at the results. Seems that implicit is worse, which is annoying/sad. Do you have any idea why oracle isn't getting 100% on the non-holding cover envs? That feels important for us to understand...

Reply:
right. and yeah, I think it is because we made the hand regions pretty unforgiving, so the low-level planning is legitimately hard in many cases. there are hyperparameters to adjust this, which we can do when we return to this environment.